ext/js/pages/permissions-main.js

```diff
@@ -102,6 +102,8 @@ await Application.main(async (application) => {
     /** @type {HTMLInputElement} */
     const permissionCheckbox2 = querySelectorNotNull(document, '#permission-checkbox-allow-file-url-access');
     /** @type {HTMLInputElement[]} */
+    // This collection is actually used, not sure why this eslint-disable is needed.
+    // eslint-disable-next-line sonarjs/no-unused-collection
```

Comment:
I don't really get this one, because the elements are used later on a few lines down.

Reply:
Maybe we should open an upstream issue for this case.

Reply:
https://github.com/SonarSource/eslint-plugin-sonarjs/issues/449

Reply:
This one seems more like a bug, at least arguably.

Thanks for creating issues, I'm happy to create any issue reports on things like this as well.